packages/core/src/components/drawer/drawer.tsx

```diff
@@ -115,7 +114,9 @@ export class Drawer extends AbstractPureComponent<IDrawerProps, {}> {
     }
 
     private maybeRenderCloseButton() {
-        if (this.props.isCloseButtonShown) {
+        // `isCloseButtonShown` can't be defaulted through default props because of props validation
+        // so this check actually defaults it to true if it is not defined (== null)
+        if (this.props.isCloseButtonShown || this.props.isCloseButtonShown == null) {
```

Comment:
you can simplify a bit with `if (this.props.isCloseButtonShown !== false)`

make sure to update the code comment too

Reply:
That was my first choice, but I saw that `== null` was already used in prop validation, so I wanted to be more consistent. No problem, I'll update it